src/Synth/ADnote.cpp

```diff
@@ -1213,21 +1213,21 @@ inline void ADnote::ComputeVoiceOscillator_LinearInterpolation(int nvoice)
 {
     for(int k = 0; k < unison_size[nvoice]; ++k) {
         int    poshi  = oscposhi[nvoice][k];
-        int    poslo  = (int)(oscposlo[nvoice][k] * (1<<24));
```

Comment:
IMO the changes in this file should have some associated comments.
It's easier to understand the 1<<24, though it's understandable if the modifications make things faster.

Reply:
I added some comments and changed the precalc value to float to prevent the implicit casting.